app-office/whid/whid-2.0.2~beta.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="Time Tracker for Freelancers and Individual Contractors"
+DESCRIPTION="What Have I Done - \
+Time Tracker for Freelancers and Individual Contractors. \
+Features: \
+* Very easy and intuitive to use  \
+* Measures time for active tasks \
+* Maintains a tree-structures of folders, customers, projects and tasks \
+* Reports time spent, breakdowns for each day in a week \
+* Reports time spent on selected items in the tree structure of tasks \
+* Stores the data in SQLite"
```

Comment:
Suggestion (looks better in HaikuDepot):
```
DESCRIPTION="What Have I Done - \
Time Tracker for Freelancers and Individual Contractors.
Features:
* Very easy and intuitive to use 
* Measures time for active tasks
* Maintains a tree-structures of folders, customers, projects and tasks
* Reports time spent, breakdowns for each day in a week
* Reports time spent on selected items in the tree structure of tasks
* Stores the data in SQLite"
```

Reply:
Note the missing backslash at the end of the lines, when using * you want to begin the next * on a new line, the backslash connects all the lines otherwise

Reply:
i see.. thanks.. updated